src/Chapter4.hs

```diff
@@ -610,7 +632,14 @@ Implement the 'Monad' instance for our lists.
 🕯 HINT: You probably will need to implement a helper function (or
   maybe a few) to flatten lists of lists to a single list.
 -}
+concatList :: List (List a) -> List a
+concatList Empty = Empty
+concatList (Cons x Empty) = x
+concatList (Cons x (Cons y ys)) = x <> y <> concatList ys
```

Comment:
I think the case with `Cons x Empty` is redundant. Your implementation is already recursive in the last pattern, so it should be enough to just handle `Empty` and `Cons x xs` patterns :+1:

Reply:
You're right! I'll fix this right away :grin: